code/src/components/ThoughtForm.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const ThoughtForm = ({ onFormSubmit, newThought, setNewThought }) => {
+	return (
+		<form className="form-container" onSubmit={onFormSubmit}>
+			<label className="main-question" htmlFor="newThought">What's making you happy right now?</label>
```

Comment:
Great that you keep the HTML label functionality so that it's accessible for screen readers.